recipe/recipe.go

```diff
@@ -1,3 +1,66 @@
+/*
+Package 'recipe' implements actions mapping to YAML recipe.
+
+Recipe syntax
+
+Recipe is composed with 3 conditional parts:
+
+- definitions and expressions for text template engine
+(https://golang.org/pkg/text/template)
+
+- header
+
+- actions
+
+Comments are allowed and should be prefixed with '#' symbol.
+
+ # Declare variable 'Var'
+ {{- $Var := "Value" -}}
+
+ # Header
+ architecture: arm64
+
+ # Actions are executed in listed order
+ actions:
+   - action: ActionName1
+     property1: true
+
+   - action: ActionName2
+     # Use value of variable 'Var' defined above
+     property2: {{$Var}}
+
+Mandatory properties for receipt:
+
+- architecture -- target architecture
+
+- actions -- at least one action should be listed
+
+Supported actions
+
+- apt -- https://godoc.org/github.com/go-debos/debos/actions#hdr-Apt_Action
```

Comment:
Can we use relative urls in some way so you don't have to hardcode godoc.org ?

Reply:
Not sure if it is possible. At least 'scheme://host' part is needed to show the link.

Reply:
saddest face, ah well